src/app/forms/input/date-picker/terra-date-picker.component.scss

```diff
@@ -99,6 +99,7 @@
     
     .selector
     {
+        width:auto !important;
```

Comment:
meine standardfrage: geht das auch ohne important?

Reply:
nein, weil feste Breiten im untercontainer (my-date-picker)vergeben wurden, auf die ich kein Zugriff habe.

Reply:
steht in der Komponente im selector-tag als style="..." Attribut

Reply:
ah ok :)